### DISCUSSION TITLE
"Sign in with Keycloak" Screen

### DISCUSSION THREAD

**Discussion by alibugatekinn** [keycloak/keycloak#33385] [Q&A]
![Screenshot from 2024-09-30 11-49-05](https://github.com/user-attachments/assets/4f13e00d-0279-413f-a023-501a77a77960)

hi everybody, we are developing a nextJS project. and we are using keycloak for auth proccess. but we are encountering a page sometimes in production mode. even if we dont have any problem, we seeing this page sometimes. and it is never closing for short time. this issue  happening while the program need on the login page. how can we fix it ?

**Comment by codespearhead** [comment]
Are you using Auth.js (formerly known as NextAuth.js) [1]? If so, could you please close this discussion [2] and create an issue in their repository instead?

Keycloak itself provides only a low-level, framework-agnostic, headless OIDC client library called keycloak.js [3], and it's up to each framework ecosystem to create a wrapper for it, usually by developing simple, framework-specific middleware on top of that library.

Even if you're not using Auth.js, the first paragraph still applies.

[1] https://github.com/nextauthjs/next-auth
[2] https://docs.github.com/en/discussions/managing-discussions-for-your-community/managing-discussions#closing-a-discussion
[3] https://www.npmjs.com/package/keycloak-js
